src/main/java/salesforce/LoginPage.java

```diff
@@ -0,0 +1,45 @@
+package salesforce;
+
+import org.openqa.selenium.By;
+import gui.InteractEnum;
+
+public enum LoginPage implements InteractEnum {
+    LOGIN_BUTTON_MAIN(By.linkText("Login"), false, ""),
```

Comment:
Using enum is a good solution,  but I think it will be better if you use @findby with PageFactory in future projects